MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -785,6 +789,78 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
                         break;
                     }
 
+                case "thumb":
+                    {
```

Comment:
Can be done in a later PR, but I don't like this switch statement being so long. It should probably be changed either to a polymorphic solution or just method calls

Reply:
I have started working on a PR to use methods and make the switch statement shorter